lib/linux_admin/service/systemd_service.rb

```diff
@@ -44,12 +37,23 @@ def restart
     end
 
     def reload
-      Common.run!(Common.cmd(:systemctl), :params => ["reload", name])
+      Common.run!(command_path, :params => ["reload", name])
       self
     end
 
     def status
-      Common.run(Common.cmd(:systemctl), :params => ["status", name]).output
+      Common.run(command_path, :params => ["status", name]).output
+    end
+
+    def show
+      output = Common.run!(command_path, :params => ["show", name]).output
+      output.split("\n").map { |line| line.split("=", 2) }.to_h
```

Comment:
Generally I would prefer `collect` over `map` (I thought we had that in the Rubocop config), but in this case I would recommend `each_with_object({})`.